beekeeper-metadata-cleanup/src/test/java/com/expediagroup/beekeeper/metadata/cleanup/service/PagingMetadataCleanupServiceTest.java

```diff
@@ -150,6 +201,42 @@ public void typicalPartitioned() {
     verifyNoMoreInteractions(pathCleaner);
   }
 
+  @Test
+  @Transactional
+  public void disabledPartitioned() {
+    when(metadataCleaner.tableHasProperty(Mockito.any(), Mockito.any(), Mockito.any(), Mockito.any(),
```

Comment:
Like you have added two enabled tables in unpartitioned test. It will be good to add a enabled table here too and assert to test that the existing functionality is not broken.

Reply:
ok, I had it like that initially but changed it cause I thought it would be more confusing